halo2_proofs/src/plonk/circuit.rs

```diff
@@ -705,7 +808,44 @@ pub enum Expression<F> {
     Scaled(Box<Expression<F>>, F),
 }
 
+
+
 impl<F: Field> Expression<F> {
+
+    /// Make side effects
+    pub fn load_to_virtual_cells(&mut self, cells: &mut VirtualCells<'_, F>) {
```

Comment:
```suggestion
    /// Queries all column cells
    pub fn query_cells(&mut self, cells: &mut VirtualCells<'_, F>) {
```
Or something like that.